src/ert/parsing/file_context_token.py

```diff
@@ -34,6 +34,17 @@ def __repr__(self) -> str:
     def __str__(self) -> str:
         return self.value
 
+    def __eq__(self, other) -> bool:
+        if isinstance(other, str):
+            return self.value == other
+        if isinstance(other, Token):
+            return self.value == other.value
+        else:
```

Comment:
i guess the `else` is not strictly necessary, as the `if`s have `return`s in them